deeprank_gnn/models/grid.py

```diff
@@ -5,31 +5,39 @@
 
 from enum import Enum
 from typing import Dict
-
 import numpy
 import h5py
-
-from deeprank_gnn.domain.storage import *
+import itertools
+from scipy.signal import bspline
+from ..domain.storage import (
+    HDF5KEY_GRID_POINTS,
+    HDF5KEY_GRID_X,
+    HDF5KEY_GRID_Y,
+    HDF5KEY_GRID_Z,
+    HDF5KEY_GRID_CENTER,
+    HDF5KEY_GRID_MAPPEDFEATURES,
+    HDF5KEY_GRID_MAPPEDFEATURESVALUE
+    )
 
 
 class MapMethod(Enum):
-    """ This holds the value of either one of 4 grid mapping methods.
-        A mapping method determines how feature point values are divided over the grid points.
+    """This holds the value of either one of 4 grid mapping methods.
+    A mapping method determines how feature point values are divided over the grid points.
     """
 
     GAUSSIAN = 1
     FAST_GAUSSIAN = 2
-    BSP_LINE = 3
+    # BSP_LINE = 3
```

Comment:
Why is BSP_LINE disabled?

Reply:
I forgot to uncomment it! I'll do it before merging